CNV-61642 - added Monitor AAQ RN (#96523)
openshift/openshift-docs: virt/release_notes/virt-4-20-release-notes.adoc

[id="virt-4-20-virtualization_{context}"]
=== Virtualization
//CNV-61642

> **Monitor AAQ resource quota usage with new metrics**

Reviewer: Please delete this line. We don't use headings for individual release notes.

> The Application Aware Quota (AAQ) controller now exposes two new Prometheus metrics: `kube_application_aware_resourcequota` and `kube_application_aware_resourcequota_creation_timestamp`. You can use these metrics to gain better visibility into the current usage and creation times of application-aware resource quotas, helping you effectively monitor and manage your OpenShift Virtualization resources.

Reviewer: ```suggestion
* You can now use the `kube_application_aware_resourcequota` and `kube_application_aware_resourcequota_creation_timestamp` metrics to query the current usage and creation times of the Application-Aware Quota (AAQ) Operator resources.
```

Link to the feature documenation from the release note after https://github.com/openshift/openshift-docs/pull/96517 is merged. As a side note, use the variable {VirtProductName} to refer to OpenShift Virtualization.

[id="virt-4-20-virtualization_{context}"]
=== Virtualization
//CNV-61642

> * You can now use the `kube_application_aware_resourcequota` and `kube_application_aware_resourcequota_creation_timestamp` metrics to query the current usage and creation times of the Application-Aware Quota (AAQ) Operator resources. For more information, see xref:../../virt/monitoring/virt-prometheus-queries.adoc#virt-querying-metrics_virt-prometheus-queries [AAQ Operator metrics].

Reviewer 2: ```suggestion
* You can now use the `kube_application_aware_resourcequota` and `kube_application_aware_resourcequota_creation_timestamp` metrics to query the current usage and creation times of the Application-Aware Quota (AAQ) Operator resources. For more information, see xref:../../virt/monitoring/virt-prometheus-queries.adoc#virt-promql-AAQ-metrics_context[AAQ Operator metrics].
```
Hopefully this gets rid of the xref errors...
